omero/sysadmins/version-requirements.txt

```diff
@@ -865,850 +611,239 @@ requires Python 2.7. The older version (1.6) will work with Python
       - Homebrew
       - FreeBSD Ports
-    * - 3.3
-      - N/A
-      - N/A
-      - 10.04
-      - 6.0
-      - N/A
-      - N/A
-    * - 3.4
-      - N/A
-      - N/A
-      - 12.04, 13.04, 13.10
-      - 7.0
-      - N/A
-      - N/A
-    * - 3.5
+    * - 7
+      - 6, 7
+      - 14.04
       - N/A
-      - 19, 20, 21
-      - 14.04, 14.10, (15.04)
-      - (8.0)
-      - Yes
       - Yes
-    * - 3.6
-      - N/A
-      - N/A
-      - N/A
-      - N/A
-      - N/A
-      - N/A
-
-Java
-----
-
-`General overview <http://www.oracle.com/technetwork/java/eol-135779.html>`__
-
-.. list-table::
-    :header-rows: 1
-
-    * - Version
-      - Release date
-      - Upstream support
-      - OMERO 5.1
-      - OMERO 5.2
-      - OMERO 5.3
-      - Details
-    * - 6
-      - from Dec 2006
-      - to Feb 2013
-      - Deprecated
-      - Dropped
-      - Dropped
-      - `Reference <http://www.oracle.com/technetwork/java/eol-135779.html>`__
-    * - 7
-      - From Jul 2011
-      - to Apr 2015
-      - Supported
-      - Deprecated
-      - Deprecated
-      - `Reference <http://www.oracle.com/technetwork/java/eol-135779.html>`__
     * - 8
-      - From Mar 2014
-      - to Mar 2017
-      - Recommended
-      - Recommended
-      - Recommended
-      - `Reference <http://www.oracle.com/technetwork/java/eol-135779.html>`__
-
-Distribution support
-^^^^^^^^^^^^^^^^^^^^
-
-.. list-table::
-    :header-rows: 1
-
-    * - Version
-      - CentOS/RHEL
-      - Fedora
-      - Ubuntu
-      - Debian
-      - Homebrew
-      - FreeBSD Ports
-    * - 6
-      - 6.x, 7.x
-      - N/A
-      - 10.04, 12.04, 13.04, 13.10, 14.04, 14.10, (15.04)
-      - 6.0, 7.0, (8.0)
+      - 6, 7
+      - 16.04
       - N/A
       - Yes
-    * - 7
-      - 6.x, 7.x
-      - 19, 20, 21, (22)
-      - 12.04, 13.04, 13.10, 14.04, 14.10, (15.04)
-      - 7.0, (8.0)
+    * - 9
+      - N/A
       - N/A
-      - Yes
-    * - 8
       - N/A
-      - 19, 20, 21, (22)
-      - 14.10, (15.04)
-      - (8.0)
       - N/A
-      - Yes
     * - Details
       - 
-      - `Ref 1 <https://apps.fedoraproject.org/packages/java-1.7.0-openjdk>`__
-        `Ref 2 <https://admin.fedoraproject.org/pkgdb/package/java-1.8.0-openjdk/>`__
-      - `Ref <http://packages.ubuntu.com/search?keywords=jdk&searchon=names&suite=all&section=all>`__
-      - `Ref <https://packages.debian.org/search?keywords=jdk&searchon=names&suite=all&section=all>`__
+      - `Reference <http://packages.ubuntu.com/search?keywords=jdk&searchon=names&suite=all&section=all>`__
       - 
       - 
 
-Note that all distributions provide OpenJDK 6, 7 and/or 8 due to
+Note that all distributions provide OpenJDK 7 and/or 8 due to
 distribution restrictions by Oracle. Oracle Java may be used if
 downloaded separately.
 
-Oracle no longer allow general downloading of Java 6. Java 7 is
-available for all platforms except for OSX 10.6 (where it is
+Java 7 is available for all platforms except for OSX 10.6 (where it is
```

Comment:
This whole paragraph looks out of date to me, not sure we need to keep any of it.